### ISSUE TITLE
The <Inventory> command from the guides.xml is not being used when rendering with Docker

### ISSUE THREAD

**Issue by FlorianT03** [TYPO3-Documentation/render-guides#96]
I have added the `<Inventory>` command in the file "guides.xml" under the path "Documentation/guides.xml" with the help of the "Settings.cfg"

Example of a line:

`<inventory id="t3coreapi11" url="https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/" />`

Then I executed the command `run --rm --pull always -v $(pwd):/project -u $(id -u):$(id -g) -it ghcr.io/typo3-documentation/render-guides:main --output Documentation-GENERATED-temp --config Documentation/ Documentation` and got many error messages such as:

[2023-11-20T10:51:23.921246+00:00] app.WARNING: Reference typo3-fluid-format-html could not be resolved in Security/GuidelinesExtensionDevelopment/Index {"rst-file": "Security/GuidelinesExtensionDevelopment/Index"}

So there must be a solution that allows the guides.xml to be read correctly again and thus also the `<inventory>` command.

**Comment by garvinhicking** [comment]
(Just to add to this before I forget - I believe Lina said that within tests the "guides.xml" of a test is properly interpreted, with their `<inventory>` tags. So there might be a different in stitching the symfony containers and decorators together, which makes it work in a test but not in the "actual" docs?)

**Comment by garvinhicking** [comment]
This should now be adressed - Florian, could you verify? The Configuration should be taken into account.

Of course other/additional warnings (due to missing rendering definitions) will stay, but at least guides.xml and inventories should now be evaluated!

**Comment by FlorianT03** [comment]
Yes I only saw the warnings from the missing rendering definitions.
